pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -1750,17 +1744,6 @@ void Object_beginTypeContext(JSOBJ _obj, JSONTypeContext *tc) {
             goto INVALID;
         }
 
-        return;
-    } else if (PyLong_Check(obj)) {
```

Comment:
Also unreachable; this used to be PyInt_Check so could yield something in Python2, but with switch to Python3 was duplicative of the PyLong_Check that immediately preceded it